vervet-underground/internal/storage/digest.go

```diff
@@ -33,8 +35,8 @@ func DigestHeader(value string) string {
 	return ""
 }
 
-// Digest returns the digest of the given contents.
-func Digest(contents []byte) string {
+// GenerateDigest returns the digest of the given contents.
+func GenerateDigest(contents []byte) Digest {
```

Comment:
This is getting more into bikeshed territory, but I wonder if this is really a constructor? I can go either way on this.

```suggestion
// NewDigest returns the digest of the given contents.
func NewDigest(contents []byte) Digest {
```